cpp/src/slice2cs/CsUtil.cpp

```diff
@@ -1705,7 +1705,7 @@ Slice::CsGenerator::writeOptionalSequenceMarshalUnmarshalCode(
                     }
                     else
                     {
-                        out << nl << "if (" << param << " is not null";
+                        out << nl << "if (" << param << " is not null)";
```

Comment:
How did this work before?

Reply:
It didn't. We didn't catch it because there was no test.